e2e/cypress/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Message Reply', () => {
+    before(() => {
+        // # Change viewport to tablet
+        cy.viewport('ipad-2');
+
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+
+        // # Post a new message to ensure there will be a post to click on
+        cy.postMessage('Hello ' + Date.now());
+    });
+
+    it('M18705-Center channel input box does not overlap with RHS', () => {
+        const maxReplyCount = 15;
+
+        // * Check if center channel post text box is focused
+        cy.get('#post_textbox').should('be.focused');
+
+        // # Click "Reply"
+        cy.getLastPostId().then((postId) => {
+            cy.clickPostCommentIcon(postId);
+        });
+
+        // * Check if center channel post text box is not focused
+        // Although visually post text box is not visible to user,
+        // cypress still considers it visible so the assertion
+        // should('not.be.visible') will fail
+        cy.get('#post_textbox').should('not.be.focused');
+
+        // # Post several replies
+        cy.get('#reply_textbox').clear().should('be.visible').as('replyTextBox');
+        for (let i = 1; i <= maxReplyCount; i++) {
+            cy.get('@replyTextBox').type(`post ${i}`).type('{enter}');
+        }
+
+        // * Check if "Add Comment" button is visible
+        cy.get('input[type=button]').scrollIntoView().should('be.visible').and('have.value', 'Add Comment');
```

Comment:
Instead of `input[type=button]` identifier, can you add the ID for the comment button (in create_comment.jsx file) and then use the same. 

Please check this [link](https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs) for more information. 

![Screenshot 2019-11-18 at 5 57 07 PM](https://user-images.githubusercontent.com/1429138/69052427-014f4600-0a2d-11ea-8e15-3b7a496df73b.png)

<img width="1054" alt="Screenshot 2019-11-18 at 5 58 42 PM" src="https://user-images.githubusercontent.com/1429138/69052456-14621600-0a2d-11ea-9113-56662c87bc3d.png">

Reply:
Updated in latest commit